### ISSUE TITLE
About the negative rewards in training process

### ISSUE THREAD

**Issue by zzhnobug** [AI4Finance-Foundation/FinRL#187]
I have tried several kinds of models such as ddpg, ppo. However, when I check the training log, It seems the agent have lost almost all initial amounts.
environment/            |             |
|    portfolio_value      | 9.33e+04    |
|    total_cost           | 1.43e+05    |
|    total_reward         | -9.07e+05   |
|    total_reward_pct     | -90.7       |
|    total_trades         | 103492      |
| time/                   |             |
|    fps                  | 95          |
|    iterations           | 40          |
|    time_elapsed         | 858         |
|    total_timesteps      | 81920       |
| train/                  |             |
|    approx_kl            | 0.062760234 |
|    clip_fraction        | 0.406       |
|    clip_range           | 0.2         |
|    entropy_loss         | -47         |
|    explained_variance   | 0.399       |
|    learning_rate        | 0.00025     |
|    loss                 | 1.04        |
|    n_updates            | 390         |
|    policy_gradient_loss | -0.0161     |
|    std                  | 1.16        |
|    value_loss           | 2.29        |
-----------------------------------------
Here is the last log when finished training, where total_reward_pct is -90.7%. Is there some bugs in calculating? Or it's just the normal performance? But the test result is normal, I am confused.
==============Start Trading===========
hit end!
==============Get Backtest Results===========
Annual return          0.064639
Cumulative returns     0.133737
Annual volatility      0.193466
Sharpe ratio           0.421550
Calmar ratio           0.299487
Stability              0.000059
Max drawdown          -0.215831
Omega ratio            1.085617
Sortino ratio          0.569701
Skew                        NaN
Kurtosis                    NaN
Tail ratio             0.866073
Daily value at risk   -0.024051
dtype: float64
